Sources/WalletConnectModal/Modal/Modal+Previews.swift

```diff
@@ -34,7 +33,7 @@ struct ModalContainerView_Previews: PreviewProvider {
         
         init() {
             
-            let projectId = "9bfe94c9cbf74aaa0597094ef561f703"
+            let projectId = Secrets.load().projectID
```

Comment:
Now we're taking Project ID from configuration. Why json file is better?

Reply:
`configuration.xcconfig` is not available from package, only for stuff in the ExampleApp project, or at least I haven't figured out how to read from it.

I'm not too keen on having two approaches how to share secrets/vars either. So any suggestions on how to do this better are more than welcome :D

Reply:
you can pass projectID to w3m in a config method like in `Relay `

```
    static public func configure(
        relayHost: String = "relay.walletconnect.com",
        projectId: String,
        socketFactory: WebSocketFactory,
        socketConnectionType: SocketConnectionType = .automatic
    )
```

Reply:
can't believe we missed comiting proj id xd

Reply:
Yeah, this is how it will be done when actually integrating the modal SDK https://github.com/WalletConnect/WalletConnectSwiftV2/blob/9cead7d4443815687e02cd87ba888130225283fc/Sources/WalletConnectModal/WalletConnectModal.swift#L51-L57C8

but this is in the context of previews within SDK itself so I don't have any projectID injected from elsewhere.